packages/fxa-content-server/tests/functional/security_events.js

```diff
@@ -0,0 +1,42 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+const { registerSuite } = intern.getInterface('object');
+const TestHelpers = require('../lib/helpers');
+const FunctionalHelpers = require('./lib/helpers');
+
+const {
+  clearBrowserState,
+  createUser,
+  fillOutSignIn,
+  openPage,
+  testElementExists,
+} = FunctionalHelpers;
+
+const config = intern._config;
+const SECURITY_EVENTS_URL = `${config.fxaContentRoot}security_events?security_events=true`;
+const PASSWORD = '12345678';
+let email;
+
+registerSuite('security_events', {
```

Comment:
At a high level, we should be testing the following

* No link when no `security_events` query param
* Link when `security_events` query param
* Testing that user can log in and event is show
* Testing that user can reset password and event is show
* Testing that user can delete events (it will fail since the js-client package needs to be republished)